### DISCUSSION TITLE
Github Login: Unable to exchange external code

### DISCUSSION THREAD

**Discussion by PaulRennerbs** [supabase/supabase#1700] [Questions] [answered]
hey there.

When using the github login via the <Auth> widget i just get redirected to my own page with the query parameters:

{
  error: 'server_error',
  error_description: 'Unable to exchange external code: <random  code>'
}

Can anyone help with that?

**Comment by MildTomato** [comment, marked as answer]
Also related to this https://github.com/supabase/supabase/discussions/1192
